data-prepper-plugins/obfuscate-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/obfuscation/action/ObfuscationActionTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.obfuscation.action;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.opensearch.dataprepper.model.record.Record;
+
+interface ObfuscationActionTest {
+    
+    default Record<Event> createRecord(String message) {
+        final Map<String, Object> testData = new HashMap<>();
+        testData.put("message", message);
+        return new Record<>(JacksonEvent.builder()
```

Comment:
Please use the new `TestEventFactory`.

```
return new Record<>(TestEventFactory.getTestEventFactory().eventBuilder(LogEventBuilder.class).withData(testData).build());
```

Here is an example of use:

https://github.com/opensearch-project/data-prepper/blob/87d93a07c8932d2acb6670525d3a2b60e3f8fb9b/data-prepper-plugins/common/src/test/java/org/opensearch/dataprepper/plugins/source/file/FileSourceTests.java#L265-L268

You will also need to add this to the Gradle file for this project:

https://github.com/opensearch-project/data-prepper/blob/8a1a41626b59b45debd77e7acecf2e6b8dec7c29/data-prepper-plugins/common/build.gradle#L25

Reply:
Adressed.